packages/scandipwa/src/component/ProductActions/ProductActions.style.scss

```diff
@@ -53,12 +53,14 @@
 
     &-ShortDescription {
         font-size: 14px;
-        line-height: 20px;
         margin-block: 16px;
+        line-height: 20px;
 
         div[itemprop="description"] li:before{
             position: static;
         }
+        
+        @include description-tag-line-height
```

Comment:
Same here

Reply:
There are more default HTML tags than used here. Also, there was no common name for this kind of tag group so to avoid confusion decided to leave as a 'description-tags'.

Reply:
Then move mixing into semantically correct component. Right now mixing name describes only specific case of use.